### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by jellea** [jellea/clojure-cup-2015#1]

**Comment by plexus** [comment]
Build a frontend for Clojars download stats. They make daily stats available as EDN files but there's no easy way yet to look at them. (trending, new and upcoming, most popular this month, etc)

**Comment by jellea** [comment]
Build a beautiful interactive version of the Clojurebridge tutorial. Complete with quil, cljs-in-cljs, codemirror (with parinfer?) et all.

**Comment by pesterhazy** [comment]
Datomic Explorer: Build a pretty web frontend for a Datomic db. You can navigate from entity to entity, query reverse references. Not unlike Datomic's Console, except nicer. Use the MusicBrainz db as a showcase.

I'm assuming it would be 50% or more front end work.

**Comment by jellea** [comment]
Thinking of making something interactive along the lines of the [book of shaders](http://patriciogonzalezvivo.com/2015/thebookofshaders/02/)
but maybe we can give it a nice social spin :) Co-learning or something?

**Comment by ctford** [comment]
Convert this to a shell application that can be configured with text/examples, so that library authors can use it to generate their own documentation. It rocks.

**Comment by jellea** [comment]
@ctford Thanks! That's a great idea, to be able to generate interactive documentation for libs. Might be Interesting to combine it with Devcards? (https://github.com/bhauman/devcards)

**Comment by ctford** [comment]
@jellea Yeah, I think DevCards is pretty cool. It could be interesting to combine the two.
